src/components/application_manager/src/commands/hmi/on_exit_application_notification.cc

```diff
@@ -62,17 +62,21 @@ void OnExitApplicationNotification::Run() {
     return;
   }
 
+  HMILevel::eType app_hmi_level =
+      application_manager_.GetDefaultHmiLevel(app_impl);
   Common_ApplicationExitReason::eType reason;
```

Comment:
@dtrunov You right, I misunderstood what was needed to be done according to requirements, I have removed notification since it is redundant also I made minor cosmetic change at the end of the method. Also I think I need to rename the branch, should I do this after the reviews, or it is better to rename it now?